run.sh

```diff
@@ -31,5 +31,7 @@ else
     PORT='8000'
 fi
 
-python3 -m poetry run gunicorn -b ${IP}:${PORT} mobsf.MobSF.wsgi:application --workers=1 --threads=10 --timeout=3600 \
+poetry run python manage.py collectstatic --noinput
+
+poetry run gunicorn -b ${IP}:${PORT} mobsf.MobSF.wsgi:application --workers=1 --threads=10 --timeout=3600 \
```

Comment:
Should have `python3 -m` as we saw issues in multiple OS or configs where poetry is not always available globally.